lib/core/include/scipp/core/element_array.h

```diff
@@ -125,7 +120,7 @@ template <class T> class element_array {
       m_data.reset();
       m_size = 0;
     } else if (new_size != size()) {
-      m_data = make_unique_for_overwrite<T[]>(new_size);
+      m_data = std::make_unique_for_overwrite<T[]>(new_size);
```

Comment:
Hooray :)

Reply:
Not supported by AppleClang yet :(
```
/Users/runner/work/1/s/lib/core/include/scipp/core/element_array.h:123:21: error: no member named 'make_unique_for_overwrite' in namespace 'std'
      m_data = std::make_unique_for_overwrite<T[]>(new_size);
```

Reply:
Turns out, not even gcc supports it officially.